modules/installation-user-defined-tags-azure.adoc

```diff
@@ -3,78 +3,16 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="installing-azure-user-defined-tags_{context}"]
-= Configuring user-defined tags for Azure
+= Configuring user-defined tags for {azure-short}
 
-In {product-title}, you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+In {product-title}, you can use tags for grouping resources and for managing resource access and cost. Tags are applied only to the resources created by the {product-title} installation program and its core operators such as Machine API Operator, Cluster Ingress Operator, Cluster Image Registry Operator. The {product-title} consists of the following types of tags:
 
-Support for user-defined tags is available only for the resources created in the Azure Public Cloud. User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} {product-version}.
+* {product-title} tags: By default, {product-title} installer attaches the {product-title} tags to the {azure-short} resources. These {product-title} tags are not accessible for the users. The format of the {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned` where `cluster_id` parameter is the value of `.status.infrastructureName` in `config.openshift.io/Infrastructure` API.
```

Comment:
```suggestion
* {product-title} tags: By default, {product-title} installer attaches the {product-title} tags to the {azure-short} resources. These {product-title} tags are not accessible to users. The format of the {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned`, where `<cluster_id>` is the value of `.status.infrastructureName` in the infrastructure resource for the cluster.
```

Reply:
Thanks Jeana! I re-read and I have couple more fixes now :D

```suggestion
* {product-title} tags: By default, {product-title} installation program attaches the {product-title} tags to the {azure-short} resources. These {product-title} tags are not accessible to the users. The format of the {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned`, where `<cluster_id>` is the value of `.status.infrastructureName` in the infrastructure resource for the cluster.
```